src/meta/src/stream/mod.rs

```diff
@@ -29,8 +29,8 @@ pub use stream_manager::*;
 
 use crate::MetaResult;
 
-/// Record vnode mapping for stateful operators in meta.
-pub fn record_table_vnode_mappings(
+/// Record internal table ids for stateful operators in meta.
+pub fn record_internal_state_tables(
```

Comment:
Can we move this function to `stream_graph.rs`? And the comments in the function body can also be cleaned.